src/components/CreateCourseRunPage/CreateCourseRunForm.jsx

```diff
@@ -93,6 +115,10 @@ const BaseCreateCourseRunForm = ({
 
 BaseCreateCourseRunForm.defaultProps = {
   currentFormValues: {},
+  courseRunLabels: [],
+  initialValues: {
```

Comment:
Why is initialValues in the defaultProps and propTypes if it isn't being used as a prop?

Ahh. I see, you're passing it in. But it's kinda handled behind the scenes. I'd remove it out of here. We don't specify initialValues in the props anywhere else.